### ISSUE TITLE
Updates?

### ISSUE THREAD

**Issue by civicdirect** [lquessenberry/OpenSocialOnPantheon#2]
Hi there?  I'm wondering if you would be incorporating some of the latest core and open social updates

**Comment by lquessenberry** [comment]
I will look into it in the next few days. I was just talking about getting back on to this project to maintain it.

**Comment by civicdirect** [comment]
Great thanks.  in my updating to 1.5 i've had some issues where l I am no longer able to create content on existing content types.  https://www.drupal.org/node/2881673  It would be great to have someone with much deeper expertise leading the way.

**Comment by lquessenberry** [comment]
This may not be the best issue tracker for your request. Hopefully the Drupal community can help. My project is merely here on github to allow installation on Pantheon.io hosting

**Comment by civicdirect** [comment]
Of course.  Just wanted to give you a heads up on what I was experiencing after doing an update to latest version.

**Comment by jaarong** [comment]
So, first of all, thanks for this.  After several tries to get Open Social working on Pantheon, this is the only thing that worked.  Of course, my problem is, this is all very outdated.  I realize you may not have time to update the project, but can you give me some tips or get me started on the right path to updating this to Open Social 1.2?

**Comment by jaarong** [comment]
I finally got this working for anyone interested.  A couple of key steps that I was missing was that I updated through composer locally using "composer update --with-dependencies" then had to remove the .git folders for flag and r4032login.

**Comment by gusaus** [comment]
Now that Pantheon [provides better support](https://pantheon.io/docs/create-custom-upstream/), an upstream with the latest release (now 1.9) would be even more valuable to users and developers Pantheon and Open Social.

With that in mind, @lquessenberry do you know if this is still the goto Open Social/Pantheon upstream? If so, do we have a sense for the time it would take to update this repo to the latest release? Would it be something you'd be interested/able to manage if you had some sort of support from the community?

**Comment by kainosnetwork** [comment]
I would love to use this upstream if we could get it updated to the most recent version. I am willing to test, or point me in the wright direction and I can try to work on the code, although I am not much of a coder.

**Comment by jas22g** [comment]
I'm  beta tester for Google play services already, but now I want participate in Android regular Oreo and upcoming Android P, for my Android one MiA1 device

**Comment by lquessenberry** [comment]
Hey guys. After a long hiatus, I am retracing my steps and I'll report back with a status on where this fix is needed in 2023. 

I apologize for any delay or inconvenience I may have caused during my radio silence.

**Comment by gusaus** [comment]
Great to hear from you @lquessenberry! There's been a good bit of recent interest ([for example](https://www.drupal.org/project/drupal_contribution/issues/3281963#comment-14811214)) in updating/maintaining/extending what you've created here. Would it be helpful if you had some additional contributors and maybe some funding for the work? 

If yes, possibly we could create or associate an [Open Collective](https://opencollective.com/) with the project?

**Comment by lquessenberry** [comment]
I would be thrilled to do this in my spare time. I am considering
contributing some efforts to create some themes for OpenSocial as well. My
background focuses on atomic design, component-based (BEM / ABEM) code
structure, and extending Twig through the components module.  I will gladly
contribute. Being a lone maintainer is often daunting but a little beer and
coffee money goes a long way. I still use Pantheon quite often for other
projects and I have a healthy relationship with many familiar faces there.
Let's do this.

Lee Quessenberry

**Comment by lquessenberry** [comment]
Gus,

Where and when is a good place to sync up? Drupal slack?

Lee

**Comment by gusaus** [comment]
Hey @lquessenberry!

Realized we haven't touched base in a bit and was just wondering how things have been progressing and if you need any help. 

With regards to the latter, I just realized that https://github.com/pantheon-systems has a bunch of upstream repos. Not sure if they help support, but it might be worth looking into?
